activerecord/test/cases/persistence_test.rb

```diff
@@ -1572,4 +1574,17 @@ def test_query_constraints_raises_an_error_when_no_columns_provided
   def test_child_class_with_query_constraints_overrides_parents
     assert_equal(["clothing_type", "color", "size"], ClothingItem::Sized.query_constraints_list)
   end
+
+  def test_update_after_becomes
+    book = Book.create!(name: "adf")
+    converted_book = book.becomes(EncryptedBookCustomType)
+    converted_book.name = "Testing"
+    converted_book.save
+    if Book.find(converted_book.id).name == "1"
+      assert_equal "1", Book.find(converted_book.id).name
+    else
+      # postgresql uses "t" instead of "1"
+      assert_equal "t", Book.find(converted_book.id).name
+    end
```

Comment:
it looks a bit overcomplicated from reading perspective, perhaps we could fetch the book once and avoid the branch alltogether
```suggestion
    name = Book.find(converted_book.id).name
    assert_includes ["1", "t"], name
```

we could leave a comment clarifying what `"t"` is but overall I think this is a reasonable and a short assertion since all we care is that the name is not `Testing` or `adf`

Reply:
I like how succinct this is.  Thank you!